.vib/flink/runtime-parameters.yaml

```diff
@@ -0,0 +1,115 @@
+jobmanager:
+  containerPorts:
+    rpc: 6555
+    http: 8078
+    blob: 9191
+  service:
+    type: LoadBalancer
+    ports:
+      rpc: 6557
+      http: 80
+      # Must be equal to the containerPort.blob. More info in the values documentation.
+      blob: 9191
+  podSecurityContext:
+    fsGroup: 1002
+  containerSecurityContext:
+    runAsUser: 1002
+taskmanager:
+  service:
+    ports:
+      rpc: 14272
+  containerPorts:
+    rpc: 5779
```

Comment:
You may remove this, there is no explicit test checking this change is effectively applied (and I don't think it is critical to the testing phase either)